node/src/components/chainspec_loader.rs

```diff
@@ -412,6 +453,42 @@ where
             Event::Request(ChainspecLoaderRequest::GetCurrentRunInfo(responder)) => {
                 responder.respond(self.get_current_run_info()).ignore()
             }
+            Event::Request(ChainspecLoaderRequest::GetChainspecRawBytes(responder)) => {
```

Comment:
`GetChainspecRawBytes` returns both chainspec bytes and genesis accounts. Is this intended?

Reply:
It should be returning the chainspec bytes, and optionally returning the genesis accounts and the global state toml

Reply:
In that case I would consider renaming the `ChainspecRawBytes` struct. Maybe `ChainspecContext` or `GenesisData`, etc.

Reply:
> Yes, @Fraser999 and I discussed this, however, we think its best to see those tests as a follow up PR

Great!
Are you going to request a ticket?

Reply:
No, because its not a "context" object, and the data isn't specific to genesis only,